streams/src/main/java/org/apache/kafka/streams/kstream/Aggregator.java

```diff
@@ -35,4 +35,11 @@
      * @return           the new aggregate value
      */
     T apply(K aggKey, V value, T aggregate);
+
+    /**
+     * Return the initial value for an aggregation.
+     *
+     * @return  the initial value for an aggregation
+     */
+    T init();
```

Comment:
`init` sounds to me like `initialisation`, but seems from the javadoc that it really means `initial`.